pkg/proc/internal/ebpf/helpers_disabled.go

```diff
@@ -0,0 +1,47 @@
+//go:build !ebpf
+// +build !ebpf
+
+package ebpf
+
+// #include "./trace_probe/function_vals.bpf.h"
```

Comment:
Do we need this? I wouldn't expect to need a C compiler if eBPF was disabled.

Reply:
We still need the definition of `C.function_parameter_list_t` for the `CreateFunctionParameterList` function.

Reply:
What if it was called internally by UpdateArgMap so it didn't need to be in the disabled version?

Reply:
That's a great refactor. Done.